addon/helpers/author-list.js

```diff
@@ -0,0 +1,30 @@
+import Ember from 'ember';
+const {
+  Helper
+} = Ember;
+
+export function authorList([ authors ]/*, hash*/, {unlinked=false}={}) {
+  let finalString = '';
+
+  authors.forEach((author, idx) => {
+    let line = '';
+    
+    if (unlinked){
+      line = author.name;
+    } else {
+      line = `<a href="${author.url}" >${author.name}</a>`;
+    }
+
+    if (idx === authors.length - 2){
```

Comment:
might be worth calling map on `authors` to get an array of strings (names or hyperlinks) and passing that array to your `list-separator` helper (which I think is a great helper)